services/server/server.go

```diff
@@ -515,6 +540,29 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 			DisableSTT:             req.DisableStt,
 			PMMPublicAddress:       req.PmmPublicAddress,
 			RemovePMMPublicAddress: req.RemovePmmPublicAddress,
+
+			EnableAlerting:              req.EnableAlerting,
+			DisableAlerting:             req.DisableAlerting,
+			RemoveEmailAlertingSettings: req.RemoveEmailAlertingSettings,
+			RemoveSlackAlertingSettings: req.RemoveSlackAlertingSettings,
+		}
+
+		if req.EmailAlertingSettings != nil {
+			settingsParams.EmailAlertingSettings = &models.EmailAlertingSettings{
+				From:      req.EmailAlertingSettings.GetFrom(),
```

Comment:
Nitpick – since we already checked that req.EmailAlertingSettings is not nil, we can access fields directly, without helper methods